README.md

```diff
@@ -322,18 +316,18 @@ In contrast, type assertions have no runtime behavior.
 
 #### Non-Nullable Assertions
```

Comment:
This whole feature is TypeScript specific. We should remove this whole section.

Reply:
As the currently suggested tentative grammar supports it and there is still room for discussion, I suggest we leave it as it is for now. However, we can address this topic in issues https://github.com/tc39/proposal-type-annotations/issues/31 and #142.